WunderSys.xml

```diff
@@ -39128,6 +39179,9 @@ function wsys.bellwortAffCount()
   if wsys.aff.generosity then count = count + 1 end
   if wsys.aff.lovers then count = count + 1 end
   if wsys.aff.pacified then count = count + 1 end
+	if wsys.aff.indifference then count = count + 1 end
```

Comment:
I didn't know this was an aff. It looks like we're missing indifference in pretty much every other place it may need to be. This is fine and doesn't need to be fixed now, more just making a note for a TODO